WalletWasabi.Fluent/Views/Wallets/VerifyRecoveryWordsView.axaml

```diff
@@ -11,22 +11,21 @@
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
              x:Class="WalletWasabi.Fluent.Views.Wallets.VerifyRecoveryWordsView">
   <controls:ContentArea Title="{Binding Title}"
-                        Caption="Please enter your wallet Recovery Words in the correct order"
                         NextContent="Verify" EnableNext="True" EnableCancel="True">
-    <DockPanel VerticalAlignment="Top">
+    <DockPanel VerticalAlignment="Top" Margin="0 20 0 20">
```

Comment:
Don't do this. If a margin is required it should be added to `ContentArea` to have it everywhere for consistency. But surely not added to each view individually.